.gitignore

```diff
@@ -0,0 +1,5 @@
+*.db
+*.sqlite3
+*.pyc
+.idea/
+venv/
```

Comment:
It's better to always end files with a newline character. You can read why this is the case [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), in the top answer.